app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  validates_presence_of :name, :address, :city, :state, :zip, :email, :password, :confirm_pass
+
+  def self.existing_email?(email)
+    User.exists?(email: email)
+  end
```

Comment:
We should remove this since we are no longer using it in the register_controller.

Reply:
This is causing of features coverage to be below 100%